includes/admin.inc

```diff
@@ -1741,6 +1742,13 @@ function islandora_solr_admin_settings_result_fields($form, &$form_state, $varia
     '#default_value' => isset($values['snippet']) ? $values['snippet'] : FALSE,
     '#description' => t('If a match is found on this field, the search term will be highlighted.<br /><strong>Note:</strong> Only text that has been both indexed and stored may be highlighted. While highlighting on non-tokenized fields is possible, the best results are achieved using tokenized fields. This checkbox may be grayed out if the Solr field cannot be highlighted.'),
   );
+  $form['options']['render_object_name'] = array(
```

Comment:
Isn't this already done here: https://github.com/Islandora/islandora_solr_search/blob/7.x/includes/admin.inc#L2059-L2065?